packages/ovh-manager/navbar/navbar-notification.service.js

```diff
@@ -0,0 +1,121 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+export default class NavbarNotificationService {
+  constructor(
+    $interval,
+    $q,
+    $translate,
+    // CloudMessage,
+    OvhApiNotificationAapi,
+    TARGET,
+  ) {
+    'ngInject';
+
+    this.$interval = $interval;
+    this.$q = $q;
+    this.$translate = $translate;
+    // this.CloudMessage = CloudMessage;
+    this.OvhApiNotificationAapi = OvhApiNotificationAapi;
+    this.TARGET = TARGET;
+
+    this.NOTIFICATION_REFRESH_TIME = 60000;
+  }
+
+  getMessages() {
+    return this.$translate.refresh()
+      .then(() => this.OvhApiNotificationAapi.query({
+        lang: this.$translate.preferredLanguage(),
+        target: this.TARGET,
+      }).$promise
+        .catch((error) => {
+          // this.CloudMessage.error({ textHtml: error.message }, 'index');
+          throw error;
+        }));
+  }
+
+  getSubLinks() {
+    return this.getMessages()
+      .then(messages => messages.map(message => this.convertSubLink(message)))
+      .catch(() => undefined);
```

Comment:
We should test if message is an array & not empty before doing a map on it :)

Reply:
If you follow my first comment, this catch is strange :p 

.catch -> this.getMessages -> getMessages can return a lot of errors, it's a bad idea to not catch correctly the error :)